doc/api/events.md

```diff
@@ -44,21 +40,8 @@ myEmitter.emit('event');
 ```
 
 Any object can become an `EventEmitter` through inheritance. The example above
-uses the traditional Node.js style prototypical inheritance using
-the `util.inherits()` method. It is, however, possible to use ES6 classes as
-well:
-
-```js
-const EventEmitter = require('events');
-
-class MyEmitter extends EventEmitter {}
-
-const myEmitter = new MyEmitter();
-myEmitter.on('event', () => {
-  console.log('an event occurred!');
-});
-myEmitter.emit('event');
-```
+uses the ES6 classes. It is however possible to use traditional Node.js style
```

Comment:
'the ES6 classes'?  I'd drop the reference to `util.inherits()` entirely, by the way.